src/app/calendar/page.tsx

```diff
@@ -1,12 +1,18 @@
 import { Metadata } from "next";
+import SampleCalendar from "./clientPage";
```

Comment:
Also I thought for a sec that this page was the client page and that code for the calendar was on the client page, maybe worth changing the name of the file?